processor/filterprocessor/README.md

```diff
@@ -202,5 +202,39 @@ processors:
           - Key: container.name
             Value: (app_container_1|app_container_1)
 ```
-
 In case the no metric names are provided, `matric_names` being empty, the filtering is only done at resource level.
+
+### Filter Spans from Traces
```

Comment:
The top of this file should be updated to indicate that the processor supports traces.  This information should probably go up there as well, alongside the required parameters for logs and metrics.

Please also include a prominent warning that using this processor can result in broken traces and that it is the operator's responsibility to ensure that it only filters out leaf spans.

Reply:
I am happy to write it, but do you really think our users wouldn't take that into consideration?  I mean beyond that when we get to more advanced sampling their heads might explode :)